pkg/components/dex/manifest.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 The Lokomotive Authors
```

Comment:
manifest.go seems like a weird file name for a file containing just a single template.

Reply:
what do you suggest?

Reply:
We use `template.go` in some places, though I feel that `templates.go` would be more appropriate. Or `values.go`?

Reply:
if we are using `template.go` we should use that